apps/drec-ui/src/apps/device-group/view/containers/card/MyDeviceGroupCardHeader/MyDeviceGroupCardHeader.tsx

```diff
@@ -52,10 +52,13 @@ export const MyDeviceGroupCardHeader: React.FC<MyDeviceGroupCardHeaderProps> = (
             </div>
             <div className={classes.attributesBlockWrapper}>
                 {groupAttributes.map((attributes, i) => (
-                    <div key={i} style={{ width: '100%' }}>
+                    <div key={`group-attributes-${i}`} style={{ width: '100%' }}>
```

Comment:
still using index in key. 
it should not be used in key event as a part of string